tsconfigdoc.json

```diff
@@ -280,6 +286,12 @@
     {
       "path": "./packages/ui-components-extension"
     },
+    {
+      "path": "./packages/user"
+    },
+    {
+      "path": "./packages/user-extension"
+    },
```

Comment:
Do these bits belong here? They are not from the original PR.

Reply:
Good catch - they comes from running the integrity check locally on my folder. I will remove them.